### ISSUE TITLE
search suggestions for http server

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#266]
In my branch search-suggestions is my attempt at making the http-server provide suggestions in the search bar.

This relies on some specific javascript code in myscript.js and mysearch-suggest.js in the attached zip
[bits.zip](https://github.com/gnosygnu/xowa/files/2537379/bits.zip)

This makes an Ajax call of the form
http://localhost/en.wikipedia.org/search/term

where 'search' is a special trigger page on the http server and 'term' is the search term
This uses the existing java code (adjusted to return a string of results)

These results are then processed by the client side javascript

(I hope this babbling make sense)

**Comment by gnosygnu** [comment]
> (I hope this babbling make sense)

Yup, makes perfect sense. Tried it out on my side, and can confirm it works well. Will incorporate it into the main branch when I return from vacation next week. Thanks!
